sdk/formrecognizer/ai-form-recognizer/src/util.ts

```diff
@@ -53,7 +56,7 @@ export const maybemap = <T1, T2>(value: T1 | undefined, f: (v: T1) => T2): T2 |
 export function makeServiceClient(
   endpoint: string,
   credential: KeyCredential | TokenCredential,
-  options: GeneratedClientOptionalParams,
+  options: GeneratedClientOptionalParams & Pick<DocumentAnalysisClientOptions, "audience">,
```

Comment:
Generally we don't prefer utility types in public API because it would introduce un-necessary complexity for customers. Since this is just internal implementation and I'm fine with this.